KsApi/fragments/CommentFragment.graphql

```diff
@@ -1,9 +1,6 @@
 fragment CommentFragment on Comment {
   author {
-    id
-    imageUrl(width: 200)
-    isCreator
-    name
+    ...UserFragment
```

Comment:
Or maybe, instead of creating a new fragment we want to leave this as-is because it's using a different image size from `UserFragment`?